### ISSUE TITLE
Does not work if jsx statements are inside the For statement body

### ISSUE THREAD

**Issue by tomitrescak** [KonstantinSimeonov/tsx-control-statements#5]
Hi, thanks for the plugin, I made it work with ttypescript and is very helpful. There is a nastu bug though, when IF, Choose .. statemens are inside the body of for statement they are not translated ;/

```ts
<For
            of={item.elements.filter(e => !!e)}
            body={(child, index) => (
              <Choose>
                <When condition={child.elements && !!child.elements.length}>
                  <OutlineFolder/>
                </When>
                <Otherwise>
                  <div>
                    <OutlineItem  />
                  </div>
                </Otherwise>
              </Choose>
            )}
          />
```

Becomes

```ts
Array.from((item.elements.filter(e => !!e)), (child, index) => (React.createElement("div", null,
                React.createElement(Choose, null,
                    React.createElement(If, { condition: child.elements && !!child.elements.length },
                        React.createElement(OutlineFolder, {  })),
                    React.createElement(Otherwise, null,
                        React.createElement("div", Object.assign({  }, filterStyle(filter, child)),
                            React.createElement(OutlineItem, { }))))))))));
```

**Comment by tomitrescak** [comment]
When jsx control syntax is used all is good

**Comment by KonstantinSimeonov** [comment]
Hi, you're right, I totally missed that case. Just published v2.17.5, which adds a fix for this + a couple of test cases based on the code you've attached to the issue. Thanks for reporting this! Feel welcome to get back at me if you experience any other difficulty or problems.

**Comment by tomitrescak** [comment]
Works well thanks!
